### ISSUE TITLE
Training details for UCF sports dataset

### ISSUE THREAD

**Issue by Kay1794** [ZhengChang467/STRPM#1]
Dear authors of STRPM，

Thank you so much for sharing this fantastic work with the community! I know you are going to release the training code soon, but I am wondering if it is possible to get some details in advance about how to train STRPM on UCF sports and match the performances in the paper. Especially, 

- The train / test split of the ucf sports dataset;
- The training hyperparameters that you think may be matter for the final performance, such as learning rate, batch size, max epoch/iteration number, etc. (I am asking for this information is because that I have tried the default learning rate & decay & iteration number with both batch sizes of 4 and 8 on my own ucf split, but fail to reproduce the similar performance.)

Any information would be highly appreciated. Thank you so much!

**Comment by ZhengChang467** [comment]
Try the splits in datasets/ucfsport/ and follow the dataloader file in data_provider, the parameter is as the default. bz=8, max_iteration=30000 Besides, I highly recommend you to retrain the model in your own splits and compare the performance in the same experimental settings.

**Comment by Kay1794** [comment]
> Try the splits in datasets/ucfsport/ and follow the dataloader file in data_provider, the parameter is as the default. bz=8, max_iteration=30000 Besides, I highly recommend you to retrain the model in your own splits and compare the performance in the same experimental settings.

Thank you very much for your prompt response :)
